services/common/src/components/forms/RenderAutoSizeField.tsx

```diff
@@ -9,36 +9,19 @@ import { FormConsumer, IFormContext } from "./FormWrapper";
 
 interface AutoSizeProps extends BaseInputProps {
   minRows?: number;
-  maximumCharacters?: number;
+  maximumCharacters: number;
```

Comment:
I decided that this should be mandatory. There's no instances where this version of the component (ie the one in common) is used and the value is not supplied.